packages/moonstone/internal/Picker/Picker.js

```diff
@@ -670,9 +697,14 @@ const PickerBase = class extends React.Component {
 	}
 
 	calcButtonLabel (next, valueText) {
+		const
+			{decrementAriaLabel, incrementAriaLabel, joined} = this.props,
```

Comment:
We can simplify this a bit using default values. Also moved the `$L` calls within the conditional to avoid the unnecessary function call for joined pickers.

```js
if (!this.props.joined) {
    const {decrementAriaLabel = $L('previous item'), incrementAriaLabel = $L('next item')} = this.props;

    return `${valueText} ${next ? incrementAriaLabel : decrementAriaLabel}`;
}
```

Reply:
I will apply this.
Thanks.